### PR TITLE
[DASH-2299] Deploy to Syncano button

### PR DESCRIPTION
**Pull request by Flixow**

### CODE REVIEW

**Comment by Flixow** [comment]
We have to discuss about way of install process after clicked 'Deploy To Syncano' button

File: src/apps/Account/AccountLogin.jsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ const AccountLogin = React.createClass({
         .please()
         .list()
         .then((instances) => {
-          if (_.some(instances, { name: lastInstanceName })) {
+          if (_.some(instances, { name: lastInstanceName }) || queryNext === '/custom-sockets-registry') {
```
Comment (patrykkopycinski) [inline src/apps/Account/AccountLogin.jsx]: it shouldn't be called `custom-sockets` just `sockets`

File: src/apps/CustomSockets/CustomSocketsList.js [hunk 2]
```diff
@@ -77,7 +77,7 @@ const CustomSocketsList = React.createClass({
           primary={true}
           columnName="CHECK_ICON"
         >
-          Custom Sockets (BETA)
+          Sockets (BETA)
```
Comment (patrykkopycinski) [inline src/apps/CustomSockets/CustomSocketsList.js]: i don't think we should use `BETA` labels at all

File: src/apps/CustomSocketsRegistry/CustomSocketsRegistry.js [hunk 3]
```diff
@@ -20,6 +23,10 @@ const CustomSocketsRegistry = React.createClass({
   componentDidMount() {
     InstancesActions.fetch();
     Actions.fetchCustomSocketsRegistry();
+
+    if (this.props.location.query.socket_url) {
```
Comment (patrykkopycinski) [inline src/apps/CustomSocketsRegistry/CustomSocketsRegistry.js]: why do you want to show the dialog?

File: src/apps/CustomSocketsRegistry/CustomSocketsRegistry.js [hunk 4]
```diff
@@ -64,12 +71,17 @@ const CustomSocketsRegistry = React.createClass({
...
-    const { children } = this.props;
+    const { children, location } = this.props;
     const { term, items, isLoading, searchClicked, changeListView, filter, filterBySyncano } = this.state;
+    const socketUrl = location.query.socket_url;
 
     return (
       <div>
-        <Helmet title="Custom Sockets Registry" />
+        <Helmet title="Sockets Registry" />
+        <CustomSocketInstallDialog
```
Comment (patrykkopycinski) [inline src/apps/CustomSocketsRegistry/CustomSocketsRegistry.js]: `SocketInstallDialog`

File: src/apps/Sockets/EmptyView.jsx [hunk 5]
```diff
@@ -71,9 +71,9 @@ const EmptyView = () => {
       {/*
```
Comment (patrykkopycinski) [inline src/apps/Sockets/EmptyView.jsx]: uncomment it

File: src/apps/Sockets/SocketsDropdown.js [hunk 6]
```diff
@@ -15,7 +15,7 @@ const SocketsDropdown = () => (
     <MenuItem style={menuItemstyle} value="schedules" primaryText="Schedules" />
     <MenuItem style={menuItemstyle} value="channels" primaryText="Real-time Channels" />
     <MenuItem style={menuItemstyle} value="hosting" primaryText="Hosting" />
-    {/* <MenuItem style={menuItemstyle} value="custom-sockets" primaryText="Custom Sockets" /> */}
+    {/* <MenuItem style={menuItemstyle} value="custom-sockets" primaryText="Sockets" /> */}
```
Comment (patrykkopycinski) [inline src/apps/Sockets/SocketsDropdown.js]: uncomment

**Pull request closed without merging**